openedx_events/enterprise/data.py

```diff
@@ -94,3 +94,80 @@ class LedgerTransaction(BaseLedgerTransaction):
     parent_content_key = attr.ib(type=str, default=None)
     fulfillment_identifier = attr.ib(type=str, default=None)
     reversal = attr.ib(type=LedgerTransactionReversal, default=None)
+
+
+@attr.s(frozen=True)
+class EnterpriseCustomerUser:
+    """
+    Attributes of an ``enterprise.EnterpriseCustomerUser`` record.
+    """
+
+    id = attr.ib(type=int)
+    created = attr.ib(type=datetime)
+    modified = attr.ib(type=datetime)
+    enterprise_customer_uuid = attr.ib(type=UUID)
+    user_id = attr.ib(type=int)
+    active = attr.ib(type=bool)
+    linked = attr.ib(type=bool)
+    is_relinkable = attr.ib(type=bool)
+    invite_key = attr.ib(type=UUID)
+    should_inactivate_other_customers = attr.ib(type=bool)
```

Comment:
The `id` corresponds to the PK of this record, while the `user_id` represents the LMS user ID. We have since adopted a pattern of naming it `lms_user_id`, but this django model was created long ago. I guess the event can name this field anything and doesn't have to match the model.

Reply:
I think I'll just keep it matching the model.